src/components/Notification/Notification.styles.tsx

```diff
@@ -0,0 +1,54 @@
+import color from "../../styles/colors";
+import fonts from "../../styles/fonts";
+import resetCSS from "../../styles/reset";
+
+const initialStyles = `
+    ${resetCSS}
+    ${fonts.text}
+    background: #112F5C;
+    border-radius: 20px;
+    color: white;
+    min-width: 270px;
+    max-width: 25%;
+    height: 84px;
+    padding: 16px;
+    display: flex;
+    align-items: center;
+    font-size: 14px;
```

Comment:
I think we need ${fonts.textSmall} now right? cuz font size 14px should be save there but this will also effect the line height, check the typography on the far left of figma and make a textSmall please

Reply:
Gotchya!